<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inter_Lagrange</class>
 <widget class="QMainWindow" name="Inter_Lagrange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDoubleSpinBox" name="x1_IL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x2_IL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x3_IL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x4_IL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x5_IL">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fx1_IL">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fx2_IL">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fx3_IL">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fx4_IL">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fx5_IL">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X_IL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>57</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QTextEdit" name="TEXTO_IL">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="EJECUTAR_IL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejecutar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RESET_IL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SALIR_IL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;INTERPOLACION DE LAGRANGE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;x=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;f(x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;f(x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;f(x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;f(x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;4&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;f(x&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; vertical-align:sub;&quot;&gt;3&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
